pages/about-us.md

```diff
@@ -0,0 +1,49 @@
+     - topic: Integrity and Ethics
+       description: As public servants, we adhere to fundamental principles of ethical behavior. In accordance with the examples set by BOEM leadership, each BOEM employee is expected to demonstrates both professional and personal integrity. This includes a commitment to the highest level of scientific and scholarly integrity.
+       image-source: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/icon-ethics.svg
+       alt-text: An icon showing a side proflie of a hand, palm-upwards, with a Scale of Justice above it.
+     - topic: Science-Informed Decisions
+       description: BOEM is committed to using the best available science in bureau decision making. To inform the wide range of decisions within the bureau, BOEM employs a significant number of scientists and technical experts across relevant disciplines to make sound decisions at all levels of the organization.
+       image-source: https://raw.githubusercontent.com/Bixal/rrt-content/main/assets/img/icon-science.svg
+       alt-text: An icon dipicting an atom. Three rings surround a small dot at the center.
+leadership:
+     - title: Director
+       name: Amanda Lefton
+       image-source: https://www.boem.gov/sites/default/files/styles/leadership/public/2021-03/LeftonAmanda_copyright-NY.jpg?itok=9LylxJ2G
+       alt-text: Headshot photo of Amanda Lefton with the American flag in the background
+     - title: Deputy Director
+       name: Walter Cruickshank, Ph.D.
+       image-source: https://www.boem.gov/sites/default/files/styles/leadership/public/2019-03/BOEM%20Dep.%20Director%20Photo.JPG?itok=K808AQi7
+       alt-text: Headshot photo of Walter Cruickshank with the American flag in the background
+program-chiefs:
+     - title: Budget & Program Coordination
+       name: James Anderson
+       image-source: https://www.boem.gov/sites/default/files/styles/leadership/public/2019-10/James%20Anderson.jpg?itok=TpwchbRD
+       alt-text: Headshot photo of James Anderson with the American flag in the background
+     - title: Stategic Resources
+       name: Dr. Megan Carr
+       image-source: https://www.boem.gov/sites/default/files/styles/leadership/public/2020-08/Megan%20Carr.jpg?itok=yqieVPfd
+       alt-text: Outdoors headshot photo of Dr. Megan Carr
+     - title: Environmental Program
+       name: William Yancey Brown
+       image-source: https://www.boem.gov/sites/default/files/styles/leadership/public/2019-10/Bill%20Brown.jpg?itok=dxGVQDCC
+       alt-text: Headshot photo of William Yancey with the American flag in the background
+source-domain: boem.gov
+source-url: https://www.boem.gov/about-boem
+figma: https://www.figma.com/file/s0zKIEPUh1k0oW4FDvVeIb/USWDS-Templates-Truss-Lib-v2.10.0?node-id=399%3A2542
+---
+
+The Mission of the Bureau of Ocean Energy Management is to manage development of U.S. Outer Continental Shelf energy and mineral resources in an environmentally and economically responsible way.
```

Comment:
We're not totally consistent on this, but I think because we just have a single line of "content" that is essentially the "description" of the page, this is a good case to use the `description` field in the front matter, similar to what we have for the Teams page. (Not to be confused with `template-description`, which is about the template itself. This one is for a content description.

Reply:
Changes made. "Content" placed in description front matter.